test_wrapper.py

```diff
@@ -217,16 +228,16 @@ def test_handle_nonexisting_file():
 
 
 def test_store_and_load_custom_array():
-    a = [[1, 2, 3, 4], [6, 7]]
```

Comment:
Here, we don't have a test for custom_shape lists and it would fail. This bug is addressed and also tested in #33 .